modules/apps/commerce/commerce-product-test/src/testFunctional/macros/json/CommerceJSONOrdersAPI.macro

```diff
@@ -661,6 +661,36 @@ definition {
 		return ${response};
 	}
 
+	@summary = "Default summary"
```

Comment:
Can you update the summary part and put a simple description of what the macro is doing?

Something like: "This macro can be used to update order via API" etc.